### PR TITLE
try to ignore AVAudioSessionErrorCodeIsBusy

### PR DESCRIPTION
**Pull request by snylonue**
the session should be deactivated, so just ignore this error.

Some references:
https://stackoverflow.com/questions/16439767/error-domain-nsosstatuserrordomain-code-560030580-the-operation-couldn-t-be-com
https://github.com/csdcorp/speech_to_text/issues/241

### CODE REVIEW

File: lib/services/playback/playback_service.dart [hunk 1]
```diff
@@ -229,10 +230,17 @@ class PlaybackService extends ChangeNotifier {
...
       // request denied
       return;
     }
+    } on PlatformException catch (e) {
+      FLog.error(text: 'Error: $e');
+      if (e.code != '560030580') {
+        rethrow;
+      }
+    }
     await PlaybackService.player.pause();
```
Comment (Yesterday17) [inline lib/services/playback/playback_service.dart:244]: 先 pause 再设置 audio session inactive 能解决这个问题吗

Comment (snylonue) [reply lib/services/playback/playback_service.dart:244]: 感觉可以
要不把两种都试试（

Comment (Yesterday17) [reply lib/services/playback/playback_service.dart:244]: 我有空的时候发一下吧，最近忙炸了

**Comment by snylonue** [comment]
Does not work.

**Pull request closed without merging**